src/PathMatch/PathMatcherInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Files\PathMatch;
```

Comment:
```suggestion
namespace Yiisoft\Files\Matcher;
```
It would be better :)